tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -245,5 +249,25 @@ public bool IsFrameworkAssembly(AssemblyReferenceInformation assembly)
                 return string.Equals(_assemblyName, assembly?.Name);
             }
         }
+
+        private class StreamFromFileAssemblyFile : IAssemblyFile
+        {
+            private readonly byte[] _bytes;
+            private readonly string _path;
+
+            public StreamFromFileAssemblyFile(string path)
+            {
+                _path = path;
+                _bytes = File.ReadAllBytes(path);
+            }
+
+            public bool Exists { get; } = true;
+
+            public string FullName => _path;
```

Comment:
Hm... this still isn't exactly what I had in mind because it still has a path.

If a goal of this interface is to allow assemblies to be loaded from byte[]s, then we should consider what to do if the user has an assembly in memory that doesn't exist on disk. That's why I thought requiring a path property might not make sense (all assemblies have some identity - a 'name' - but not all assemblies necessarily have a path).

I suspect, though, that we still need a path for our MetadataReader interactions. If that's the case, let's just keep calling this API 'path'. But, we should think about another work item down the road to enable loading an assembly that came from some other location (web service call? reflection.emit? compiled by Roslyn but not emitted?).